### PR TITLE
Multinode Magnum missing changes

### PR DESCRIPTION
**Pull request by darmach**
Added missing routing from controllers back to seed node. Enabled Magnum, added config.

### CODE REVIEW

File: etc/kayobe/environments/ci-multinode/inventory/group_vars/controllers/network-interfaces [hunk 1]
```diff
@@ -12,6 +12,10 @@ tunnel_interface: "{{ vxlan_interfaces[0].device }}.{{ tunnel_vlan }}"
 
 public_interface: "{{ vxlan_interfaces[0].device }}.{{ public_vlan }}"
 
+public_routes:
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/inventory/group_vars/controllers/network-interfaces:15]: So this is to allow magnum to access floating IPs to communicate with k8s clusters?

File: etc/kayobe/environments/ci-multinode/inventory/group_vars/controllers/network-interfaces [hunk 2]
```diff
@@ -12,6 +12,10 @@ tunnel_interface: "{{ vxlan_interfaces[0].device }}.{{ tunnel_vlan }}"
...
+  - cidr: "{{ external_cidr }}"
+    gateway: "{{ public_allocation_pool_start }}"
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/inventory/group_vars/controllers/network-interfaces]:
```suggestion
    gateway: "{{ public_net_name | net_ip('seed') }}"
```

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: etc/kayobe/environments/ci-multinode/inventory/group_vars/controllers/network-interfaces (hunk 1 above)
Comment (darmach) [reply etc/kayobe/environments/ci-multinode/inventory/group_vars/controllers/network-interfaces:15]: Without this heat container agent was failing to grab token off keystone endpoint running on 192.168.39.2 - public network 192.168.39.0/24. It was reaching keystone fine, but replies weren't going back to external network where master was (source).

File: etc/kayobe/environments/ci-multinode/inventory/group_vars/controllers/network-interfaces (hunk 2 above)
Comment (markgoddard) [reply etc/kayobe/environments/ci-multinode/inventory/group_vars/controllers/network-interfaces]: How about this? Trying to remove the assumption that the seed has the first IP in the pool

Comment (darmach) [reply etc/kayobe/environments/ci-multinode/inventory/group_vars/controllers/network-interfaces]: Yes, it absolutely makes sense and is a more elegant choice :) I just wanted to test this version, and didn't want to risk spawning new stuff during current workshops.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**